Sources/PointFree/Episode/Episode.swift

```diff
@@ -37,37 +37,51 @@ let episodeView = View<RequestContext<Episode>> { ctx in
         ]),
 
       body(
-        [`class`([Class.pf.colors.bg.dark])],
         [
           gridRow([
             gridColumn(
-              sizes: [.xs: 12, .md: 7],
+              sizes: [.mobile: 12, .desktop: 7],
               leftColumnView.view(ctx.data)
             ),
 
             gridColumn(
-              sizes: [.xs: 12, .md: 5],
-              [`class`([Class.grid.first(.xs), Class.grid.last(.md)])],
+              sizes: [.mobile: 12, .desktop: 5],
+              [`class`([Class.pf.colors.bg.dark, Class.grid.first(.mobile), Class.grid.last(.desktop)])],
               [
                 div(
-                  [`class`([Class.position.sticky(.md), Class.position.top0])],
+                  [`class`([Class.position.sticky(.desktop), Class.position.top0])],
                   rightColumnView.view(ctx.data)
                 )
               ]
             ),
 
             ])
-          ] <> footerView.view(unit))
+          ]
+          <> downloadsAndCredits.view((codeSampleDirectory: ctx.data.codeSampleDirectory, forDesktop: false))
+          <> footerView.view(unit))
       ])
     ])
 }
 
+private let downloadsAndCredits = View<(codeSampleDirectory: String, forDesktop: Bool)> {
```

Comment:
this is a bit messy. the "downloads and credits view" is visible in the right column on desktop, but needs to be down at the bottom with the footer on mobile.

the way I did that was to render that view TWICE, with some hide classes to control when it shows :/ not great, but the way the grid is set up I dont see a clever way of swapping the order of the grid columns to get what we want.